app/views/play/level/tome/SpellView.coffee

```diff
@@ -988,6 +993,37 @@ module.exports = class SpellView extends CocoView
     @spell.transpile()  # TODO: is there any way we can avoid doing this if it hasn't changed? Causes a slight hang.
     @updateAether false, false
 
+  onWebDevError: (error) ->
+    annotations = @aceSession.getAnnotations()
+    offsetError = _.merge {}, error, { line: error.line + @linesBeforeScript(@getSource()) }
+    problem = new Problem({ error: offsetError, @ace, levelID: @options.levelID })
+    if _.any(@problems, (preexistingProblem) -> problem.isEqual(preexistingProblem))
+      problem.destroy()
+      return
+
+    @problems.push problem
+
+    lineOffsetPx = 0
+    for i in [0...problem.row]
+      lineOffsetPx += @aceSession.getRowLength(i) * @ace.renderer.lineHeight
+    lineOffsetPx -= @ace.session.getScrollTop()
+    Backbone.Mediator.publish 'tome:show-problem-alert', problem: problem, lineOffsetPx: Math.max lineOffsetPx, 0
+
+    # @saveUserCodeProblem(aether, aetherProblem) # TODO: Enable saving of web-dev user code problems
+    annotations.push problem.annotation if problem.annotation
+    @reallySetAnnotations annotations
+    Backbone.Mediator.publish 'tome:problems-updated', spell: @spell, problems: @problems, isCast: true
+
+    # TODO: Unset this on rerun
+    @ace.setStyle 'user-code-problem'
+    # TODO: When is this unset?
+    @ace.setStyle 'spell-cast'
```

Comment:
Not sure it means anything here, because of the live-coding; would have to check what the style does. But probably.